src/theme-chakra/theme/semanticTokens.ts

```diff
@@ -0,0 +1,16 @@
+export const semanticTokens = {
+  colors: {
+    text: {
+      default: '#272727',
+      _dark: '#F0F0F0',
+    },
+    background: {
+      default: '#FFFFFF',
+      _dark: '#181a1b',
+    },
+    grayShade3: {
+      default: '#F5F5F5',
+      _dark: '#1e2021',
+    },
```

Comment:
hey @ehsan-github wdyt about semantic tokens instead of `primary._light` and `primary._dark`?
I think it's just cleaner this way.

Reply:
it's a good idea!